pulser-core/pulser/backend/remote.py

```diff
@@ -81,14 +99,14 @@ def __init__(
         job_ids: list[str] | None = None,
```

Comment:
why not call the submission_id argument batch_id ?

Reply:
Because I'm afraid someone is calling `RemoteResults(submission_id=...)` and I don't want to break it